ipn/conffile/serveconf.go

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
This code is in `ipn/conffile` because I want `ServicesConfigFile` to go into `ipn.ConfigVAlpha` after this merges